src/app/(proper_react)/(redesign)/Shell/Shell.module.scss

```diff
@@ -18,7 +18,8 @@
       display: flex;
       align-items: center;
       padding: tokens.$spacing-xl tokens.$spacing-2xl;
-      border-bottom: 1px solid tokens.$color-grey-10;
+      border-bottom: 1px solid tokens.$color-grey-20;
+      box-shadow: tokens.$box-shadow-xs;
```

Comment:
This also takes effect for the existing shell. I think it's intended to go with the Toolbar changes, which also seem to have been implemented for both the existing and redesigned shell?

Reply:
I’ve reverted the changes that affect the existing shell without the feature flag enable in https://github.com/mozilla/blurts-server/pull/5671/commits/36f2f1428ab40c57c131a84534b9abb3e6972271.